admin.ghactivity.php

```diff
@@ -322,3 +332,71 @@ function ghactivity_do_settings() {
 	</div><!-- .wrap -->
 	<?php
 }
+
+/**
+ * GitHub Label Scan section.
+ *
+ * @since 2.1.0
+ */
+function ghactivity_label_scan_callback() {
+	echo '<p>';
+	esc_html_e( 'This button will scan for all the label updates for recorded issues' );
+	echo '</p>';
+
+	echo '<div class="wrap">';
+	echo '<button onclick="triggerLabelScan()">Trigger Label Rescan</button>';
+	echo '</div>';
+
+	?>
+		<script type="text/javascript" >
+		function triggerLabelScan($) {
+			if ( ! confirm( 'This is time-consuming operation. Make sure not to run it more then once!' ) ) {
+				return;
+			}
+			var data = {
+				'action': 'label_scan_action',
```

Comment:
Should we also pass a nonce here, and check it in the `label_scan_action` function with `check_ajax_referer`, to make sure no one abuses this?